distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/PaxExamRuleTest.java

```diff
@@ -197,7 +195,7 @@ public void failingAfterExam() {
             .stream()
             .anyMatch(failure -> failure.getMessage().contains(EXPECTED_AFTER_EXAM_ERROR_MESSAGE));
 
-    assertThat(examFail && afterExamFail);
+    assertTrue(examFail && afterExamFail);
```

Comment:
This change goes against our unit testing guidelines.  https://codice.atlassian.net/wiki/display/DDF/Unit+Testing